### PR TITLE
Add journalctl-mode to systemd layer

### PR DESCRIPTION
**Pull request by BenediktBroich**
This layer adds support for viewing log files with =journalctl= in Spacemacs. This is a major-mode for emacs to view systemd's journalctl output in emacs. The output is split into chunks for performance reasons. Fontification is provided and may be customized.

### CODE REVIEW

**Comment by tko** [comment]
Seems like this would fit in the existing `systemd` layer?

**Comment by BenediktBroich** [comment]
I can change that. Are the Keybinding ok?

File: layers/+tools/systemd/README.org [hunk 1]
```diff
@@ -25,7 +29,18 @@ To use this contribution add it to your =~/.spacemacs=
...
-| Key binding | Description                                                              |
-|-------------+--------------------------------------------------------------------------|
-| ~SPC m h d~ | Systemd open the directives documentation                                |
-| ~SPC m h o~ | Systemd open documentation (if a =\=Documentation= directive is provided |
+| Key binding   | Description                                                              |
+|---------------+--------------------------------------------------------------------------|
+| ~SPC m h d~   | Systemd open the directives documentation                                |
+| ~SPC m h o~   | Systemd open documentation (if a =\=Documentation= directive is provided |
+
+* Journalctl-mode
```
Comment (lebensterben) [inline layers/+tools/systemd/README.org:37]: These should be added under the `features` section above.

File: layers/+tools/systemd/README.org [hunk 2]
```diff
@@ -25,7 +29,18 @@ To use this contribution add it to your =~/.spacemacs=
...
+This is a major-mode for emacs to view systemd's journalctl output in emacs. The output is split into chunks for performance reasons. Fontification is provided and may be customized.
+
+** Key Bindings
```
Comment (lebensterben) [inline layers/+tools/systemd/README.org:40]: the key bindings for journalctl-mode should be added under the `* Key bindings` section

File: layers/+tools/systemd/packages.el [hunk 3]
```diff
@@ -40,4 +41,16 @@
...
+  (use-package journalctl-mode
+    :ensure t
+    :init (progn
+            (spacemacs/declare-prefix "atj"  "journalctl")
+            (spacemacs/set-leader-keys
+              "atjj" 'journalctl
+              "atjs" 'journalctl-unit
+              "atju" 'journalctl-user-unit
+              "atjb" 'journalctl-boot
+              ))))
```
Comment (lebensterben) [inline layers/+tools/systemd/packages.el:54]: closing parentheses should not take a new line.

**Review by lebensterben — CHANGES_REQUESTED** [review]

File: layers/+tools/systemd/README.org [hunk 4]
```diff
@@ -15,7 +17,9 @@ This layer adds support for editing =systemd= configuration files in Spacemacs.
 - Syntax highlighting
 - Auto completion
 - Syntax checking via =systemd-analyze=
-
+- Viewing systemd unit via [[https://github.com/SebastianMeisel/journalctl-mode][journalctl-mode]]
```
Comment (lebensterben) [inline layers/+tools/systemd/README.org:20]: Can you combine this two lines?

> Viewing systemd system and user unit via [[https://github.com/SebastianMeisel/journalctl-mode][journalctl-mode]]

**Review by lebensterben — APPROVED** [review]
LGTM

**Comment by lebensterben** [comment]
Merged. Thanks for your contributions.

File: layers/+tools/systemd/packages.el [hunk 5]
```diff
@@ -40,4 +40,15 @@
               "hd" 'systemd-doc-directives
               "ho" 'systemd-doc-open)))
 
+(defun systemd/init-journalctl-mode ()
+  (use-package journalctl-mode
+    :ensure t
```
Comment (arifer612) [inline layers/+tools/systemd/packages.el:45]: What's the purpose of having `:ensure t` instead of deferring it like all the other packages?

**Pull request merged**